roles/seldon_core_analytics/defaults/main.yaml

```diff
@@ -5,4 +5,5 @@ seldon_core_analytics_values: {}
 seldon_core_analytics_wait_for_deployments: true
 seldon_core_analytics_istio_external_ip_required: false
 
+seldon_core_analytics_extra_files: []
```

Comment:
:wrench: Will update to use a single list of resources, rather than using two separate lists

Reply:
The one thing I'm wondering @RafalSkolasinski is how to handle compatibility across versions - users specifying `seldon_core_analytics_extra_templates` currently will find things break, or probably just stop working.

Would it be better to explicitly transition across, e.g. with a task that does something like this:
```yaml
seldon_core_analytics_extra_resources: []

...

name: "Migrate templates to resources"
set_fact:
  seldon_core_analytics_extra_resources: "{{ seldon_core_analytics_extra_resources + [lookup('file', item) | from_yaml] }}"
with_items: "{{ seldon_core_analytics_extra_templates }}"
```

Reply:
So... I suggest we put it in "release notes" (we should start a file in the repo here if not exist)... 

In master of deploy we will use from now on just SHA of tested HEAD from this repo. And once we release 0.6 this will be indeed a breakage of backward compatibility from 0.5 to 0.6 which is fine as in `0.X` it is always allowed.

Reply:
Alternatively... Leave the other code block that depends on existing variable and add deprecation warning. 
Worth checking if ansible roles have an mechanism for it?